mage_ai/api/operations/base.py

```diff
@@ -284,6 +285,25 @@ def __payload_for_resource(self):
                 if type(payload[key]) is str:
                     payload[key] = dateutil.parser.parse(payload[key])
 
+        cookie_names = self.__resource_class().cookie_names
```

Comment:
It'll be good if you can add some comments for the code of populating cookies into payload.

Reply:
I added comments, in BaseOperation and also in the BaseResource on how to use